jobs/fxci-taskcluster-export/fxci_etl/console.py

```diff
@@ -1,9 +1,11 @@
+from datetime import datetime, timedelta
 import sys
 from pathlib import Path
 
 from cleo.application import Application
 from cleo.commands.command import Command
 from cleo.helpers import option
+import pytz
```

Comment:
sort-of unrelated nit: is there a specific reason we use the third party `pytz` package?
`datetime` has a module called `timezone` ([ref](https://docs.python.org/3/library/datetime.html#timezone-objects)). ie: `datetime.now(timezone.utc)`

Reply:
Ah thanks for calling that out.. I guess I was following some old stack overflow suggestions.

Reply:
Fixed in a new commit.